cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -327,7 +327,7 @@ macro(configure_sdk_unix name architectures)
         endif()
 	set(SWIFT_SDK_WASI_ARCH_wasm32_PATH "${SWIFT_WASI_SDK_PATH}/share/sysroot")
         # fixme: Wasi is wasm32-unknown-wasi-musl. This LLVM doesn't have it yet.
-        set(SWIFT_SDK_WASI_ARCH_wasm32_TRIPLE "wasm32-unknown-unknown-wasi")
+        set(SWIFT_SDK_WASI_ARCH_wasm32_TRIPLE "wasm32-unknown-wasi-unknown")
```

Comment:
I believe the wasi triple is just `wasm32-unknown-wasi`. No need for the extra "unknown" environment flag (I Googled it, and it seems nobody else writes it like that).

Possibly a minor issue -- but who knows? Non-exact triple matches can be a bit fragile in my experience.